contrib/opencensus-ext-azure/tests/test_azure_log_exporter.py

```diff
@@ -207,3 +213,37 @@ def test_log_with_invalid_custom_properties(self, requests_mock):
 
         self.assertFalse('not_a_dict' in post_body)
         self.assertFalse('key_1' in post_body)
+
+    @mock.patch('requests.post', return_value=mock.Mock())
```

Comment:
Surprising to see `post` patched here, but the only easy alternative I see is `TransportMixin._transmit`, which isn't obviously a better target.